### ISSUE TITLE
requireNativeComponent for Panaroma/Video Not found

### ISSUE THREAD

**Issue by kapilkarda** [tiero/react-native-360#26]
![screenshot 2019-02-02 at 5 54 48 pm](https://user-images.githubusercontent.com/4336836/52164243-1b28d200-2714-11e9-8a7b-0284e65b0cc7.png)

I am trying to use this plugin to achieve something, I am facing this issue, I have installed react-native-360 and react-native-video both the plugins and tried alone as well, but it is not working, can anyone in this community help me, I am struggling with this issue from last 5days.

Thanks in advance

**Comment by MKDhakad** [comment]
Yes, i have got same error as above , any one have solution for that . please reply

**Comment by tiero** [comment]
Couple of things, try it pulling via github as stated in the Readme and not npm and have you moved the ObjectiveC code from node_module into xcode? 
```Drag the folder RNGoogleVR from node_modules/react-native-360/ios under your myProject and be sure to check Copy items if needed and Create groups```

Also, the last version of GVRSDK breaks everything with duplicated symbols

Is tracked here https://github.com/googlevr/gvr-ios-sdk/issues/259

I will take some spare time to bring it stable if I can manage it.

**Comment by MKDhakad** [comment]
I  do that things  as you mention above your  procedure but nothing happened ...

**Comment by MKDhakad** [comment]
hay i goy same error and also "react-native camera" plugin is support "44.0" verson, so any solution to run this on 58.0 or upper versons. please revert

**Comment by kapilkarda** [comment]
I have tried the solution @tiero but I can't find its working, please help me with the same.

**Comment by haider792** [comment]
check readme of the project you will get the solution.
